modules/notifications/notifications.php

```diff
@@ -446,6 +447,7 @@ function ajax_save_post_subscriptions() {
 				
 				wp_send_json_success( $json_success );
 			}
+			remove_filter( 'ef_notification_auto_subscribe_current_user' );
```

Comment:
We need to specify which filter to remove (and the priority needs to match):

```suggestion
			// Remove auto-subscribe prevention behavior from earlier.
			remove_filter( 'ef_notification_auto_subscribe_current_user', '__return_false', PHP_INT_MAX );
```